leapp/cli/report/__init__.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import print_function
+import os
+import sys
+
+from leapp.exceptions import CommandError
+from leapp.utils.clicmd import command, command_opt, command_arg
+from leapp.utils.report import fetch_upgrade_report_messages
+from leapp.cli.upgrade import fetch_last_upgrade_context
+
+
+"""
```

Comment:
A module doc string needs to be on the top of the module.

Reply:
Sure, got confused by: 

SyntaxError: from __future__ imports must occur at the beginning of the file

but then one see:

https://docs.python.org/2/reference/simple_stmts.html#future